replications/internal/queue_management.go

```diff
@@ -286,10 +295,12 @@ func (qm *durableQueueManager) StartReplicationQueues(trackedReplications map[pl
 			continue
 		} else {
 			qm.replicationQueues[id] = &replicationQueue{
```

Comment:
Self-review: Change to use a constructor for a replication queue, since not providing the metrics field will result in a panic.